applications/nrf5340_audio/README.rst

```diff
@@ -1227,6 +1247,42 @@ Testing the BIS mode is identical to `Testing the default CIS mode`_, except for
 
 .. _nrf53_audio_app_porting_guide:
 
+Testing FOTA upgrades
+---------------------
+
+`nRF Connect Device Manager`_ can be used for testing FOTA upgrades.
+The procedure for upgrading the firmware is identical for both headset and gateway firmware.
+You can test upgrading the firmware on both cores at the same time on a headset device by completing the following steps:
+
+1. Make sure you have :ref:`configured the application for FOTA <nrf53_audio_app_configuration_configure_fota>`.
+#. Install `nRF Connect Device Manager`_ on your Android or iOS device.
+#. Connect an external flash shield to the headset.
+#. Make sure the headset runs a firmware that supports DFU using external flash memory.
+   One way of doing this is to connect the headset to the USB port, turn it on, and then run this command:
+
+   .. code-block:: console
+
+      python buildprog.py -c both -b debug -d headset --pristine -m external -p
+
+   .. note::
+      When using the FOTA related functionality in the :file:`buildprog.py` script on Linux, it is required that the ``python`` command executes Python 3.
```

Comment:
```suggestion
      When using the FOTA related functionality in the :file:`buildprog.py` script on Linux, the ``python`` command must run Python 3.
```

Reply:
Changed this sentence based on your suggestion